docs/sources/review/lint-prose/dictionary.md

```diff
@@ -0,0 +1,66 @@
+---
+description: Learn about the Grafana Labs dictionary template format.
+menuTitle: Dictionary
+review_date: 2024-06-04
+title: Grafana Labs' dictionary
+---
+
+# Grafana Labs' dictionary
+
+The Grafana Labs documentation squad maintains a dictionary used for spell checking.
```

Comment:
Do we still refer to ourselves ans a "squad"?  I remember some changes that removed the squad terminology.

Reply:
Big catch! My brain wasn't thinking. I'll update the linter to catch this and not just "docs squad"